File: week9/solution_so.js
function ht(nums) {
  var counter = 0;
  nums.forEach( num => counter ^= num );
  return counter;
}

var assert = require('assert');
var nums = [3, 5, 6, 3, 2, 2, 8, 6, 5, 5, 5];
assert.equal(ht(nums), 8)
var nums = [6, 6, 9, 7, 0, 0, 9];
assert.equal(ht(nums), 7)

File: week9/solution_so.go
package main;
import "fmt"

func ht(nums [11]int) int {
  var counter int = 0;
  for _, num := range nums {
    counter ^= num
  }
  return counter;
}

func main() {
  nums := [11]int{3, 5, 6, 3, 2, 2, 8, 6, 5, 5, 5}
  fmt.Printf("%d\n", ht(nums));
}